### PR TITLE
Add dApp-to-dApp cases

### PR DESCRIPTION
**Pull request by xrtm000**

### CODE REVIEW

File: dApp-to-dApps/add-pool-liquidity/staker.ride [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ let usdN = base58''
+
+ @Callable(i)
+ func stake() = {
+   if (i.payments[0].assetId != usdN)
+     then
+       throw("unexpected asset")
+     else {
+       let currentLiquidity = this.getInteger(i.originCaller.toString()).valueOrElse(0)
+       [ IntegerEntry(i.caller.toString(), currentLiquidity + i.payments[0].amount) ]
```
Comment (DrBlast) [inline dApp-to-dApps/add-pool-liquidity/staker.ride]: i.originCaller

File: dApp-to-dApps/swap/staker.ride [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+ let usdN = base58''
+
+ @Callable(i)
+ func stake() = {
+   if (i.payments[0].assetId != usdN)
+     then
+       throw("unexpected asset")
+     else {
+       let currentLiquidity = this.getInteger(i.caller.toString()).valueOrElse(0)
```
Comment (DrBlast) [inline dApp-to-dApps/swap/staker.ride]: i.originCaller?

**Review by DrBlast — CHANGES_REQUESTED** [review]

File: dApp-to-dApps/add-pool-liquidity/staker.ride (hunk 1 above)
Comment (xrtm000) [reply dApp-to-dApps/add-pool-liquidity/staker.ride]: Done

File: dApp-to-dApps/swap/staker.ride (hunk 2 above)
Comment (xrtm000) [reply dApp-to-dApps/swap/staker.ride]: Done

**Review by msmolyakov — APPROVED** [review]

**Review by KardanovIR — APPROVED** [review]

**Pull request merged**